.github/workflows/ci.yml

```diff
@@ -24,3 +43,21 @@ jobs:
           sudo apt install vim
       - name: Run tests
         run: make test
+
+  # Vim script linting
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
```

Comment:
Any reason this isn't newer (e.g., 3.9 or 3.10)?

Reply:
Current latest version on GitHub's `ubuntu-latest` runners for Python is 3.8.1-1, so higher than 3.8 we can't really go. See also https://packages.ubuntu.com/focal/python3

Reply:
Actually we can remove this entirely, as Python 3.8.10 comes preinstalled on `ubuntu-latest` runners.